eng/common/scripts/Update-DocsMsPackages.ps1

```diff
@@ -0,0 +1,21 @@
+# This script is intended to update docs.ms CI configuration (currently supports Java, Python, C#, JS) in nightly build
+# For details on calling, check `docindex.yml`. 
+
+# In this script, we will do the following business logic.
+# 1. Filter out the packages from release csv file by `New=true`, `Hide!=true`
+# 2. Compare current package list with the csv packages, and keep them in sync. Leave other packages as they are.
+# 3. Update the tarage packages back to CI config files. 
+param (
+  [Parameter(Mandatory = $true)]
+  $DocRepoLocation # the location of the cloned doc repo
+)
+
+. (Join-Path $PSScriptRoot common.ps1)
+
+if ($UpdateDocsMsPackagesFn -and (Test-Path "Function:$UpdateDocsMsPackagesFn")) {
+  &$UpdateDocsMsPackagesFn -DocsRepoLocation $DocRepoLocation
```

Comment:
Lets make sure we end up with the proper failure exits if something goes wrong in the processing.